metrics/scaling/workloads/bb_workload_default.cmd

```diff
@@ -0,0 +1,2 @@
+# commands in bb_*.cmd are directly used as yaml/json workload
```

Comment:
I'm going to leave this comment here, but it really covers the whole patch. But my main concern is essentially about the following three files.

So, the goal of our work is to do benchmarking in a reproducible manner consistent manner. I like the content of this PR, but what I think needs to happen is splitting your two new workloads into separate scripts. For scaling, we will want to run 
  scaling/k8s_scaling.sh   # this will generate the theoretical max pods with the minimal load
  scaling/k8s_parallel.sh   # this is the existing parallel pod launch test with minimal workload
  scaling/k8s_scaling_nc.sh   # this will reuse the scaling pod mechanism, but measuring the nc results as well
  scaling/k8s_scaling_net.sh  # this will reuse the scaling pod mechanism, but generate the net results as well

eventually, we'll be testing non-scaling (not maxing the pod counts) and we'll have more directories in addition to scaling, so we might have:
  scaling/
  scaling/k8s_scaling.sh
  scaling/k8s_parallel.sh
  scaling/k8s_scaling_nc.sh
  scaling/k8s_scaling_net.sh
  net/
  net/iperf.sh
  net/net_latency.sh
  net/net_memory.sh
  net/nginx.sh
  storage/
  storage/fio.sh
  storage/pmem.sh
  max_util/
  max_util/memory_density.sh
  max_util/cpu_density.sh

But, we want the same script to measure the same things each time. Hopefully, we're making improvements in the underlying code bases (kubernetes, kata, containerd) and through configuration changes. And we'll be able to demonstrate those improvements by running these test suites.

So, I'm thinking that what you have here is great. We just need to reorganize it a bit. The benchmarks or tests should be split up by workload. Then we have some common measures (stats.yaml) that we will take for all workloads. Hopefully this makes sense? Ideally we can reuse much of the same bash components by moving some parts out into a common scaling.bash file, that all the scaling tests can reuse.